CMakeLists.txt

```diff
@@ -172,7 +172,9 @@ if(
   )
 )
   # Attempt to auto-config
-  if(UNIX)
+  if(APPLE)
+    set(CPPTRACE_GET_SYMBOLS_WITH_LIBDL ON)
```

Comment:
Is there a reason for prioritizing libdl over libbacktrace here?

Reply:
Hmm, no. I only knew it was the only thing I know that works. I'll re-add the check for that.